modules/infra/cli/port.c

```diff
@@ -17,6 +17,8 @@
 
 static void port_show(const struct gr_api_client *, const struct gr_iface *iface) {
 	const struct gr_iface_info_port *port = (const struct gr_iface_info_port *)iface->info;
+
+	printf("driver:  %s\n", port->driver_name);
```

Comment:
Can you move it just below devargs? It seems more natural.

Reply:
done